systems/controllers/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ if (eigen3_FOUND)
 
   pods_find_pkg_config(terrain-map)
 
+  pods_find_pkg_config(lcm)
```

Comment:
it doesn't seem to be: if I remove that line, then my c++ lcm headers aren't found